src/ExamplePairing/ExamplePairing.tsx

```diff
@@ -6,6 +6,8 @@ import ReactMarkdown from "react-markdown";
 import { MethodObject, ExampleObject, ExamplePairingObject } from "@open-rpc/meta-schema";
 import _ from "lodash";
 
+export type TParamStructure = "either" | "by-name" | "by-position";
```

Comment:
I'm assuming type `string` is inferred here?

Reply:
good q.

Reply:
yep

Reply:
affirmative